examples/js/getting-started/src/app.js

```diff
@@ -1,27 +1,39 @@
-import { liteClient as algoliasearch } from 'algoliasearch/lite';
+// import { liteClient as algoliasearch } from 'algoliasearch/lite';
+import { compositionClient } from '@algolia/client-composition';
```

Comment:
as we're not really using composition yet, this PR can be done without the dependency, and we use the `runComposition` method in a separate PR?

Reply:
I 100% agree since `@algolia/client-composition` is not public anyway